src/package/nuspec/TestPlatform.Build.nuspec

```diff
@@ -14,7 +14,7 @@
     <copyright>© Microsoft Corporation. All rights reserved.</copyright>
   </metadata>
   <files>
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\Microsoft.TestPlatform.Build.dll" target="lib\netstandard1.3\" />
-    <file src="Microsoft.TestPlatform.Build\netstandard1.3\Microsoft.TestPlatform.targets" target="runtimes\any\native" />
+    <file src="Microsoft.TestPlatform.Build\Debug\netstandard1.3\Microsoft.TestPlatform.Build.dll" target="lib\netstandard1.3\" />
```

Comment:
@codito Actually this wouldn't break official nuget packages - it still gets dropped in lib\netstandard1.3 inside the folder.
I might be confused about https://github.com/Microsoft/vstest/pull/552/files#diff-56d541dcd7f193797f0c7bc290c82117L354, are the files not supposed to go in separate folders based on framework ( a netstandard one and a net46 one) ? Because it seemed as though those folders weren't created according to the comment above it. I'm happy to have the linux packages do the same, just wanted to double check first.